app/resources/stat_resource.rb

```diff
@@ -0,0 +1,13 @@
+class StatResource < BaseResource
```

Comment:
should this be `immutable`, now that I think about it `recalculate!` is run with a rake task and both `increment` + `decrement` are just inside the LibraryEntry.